### PR TITLE
Dead chat is now togglable by admins.

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Admins can now toggle dead chat.

Buildmode now has an offer control mode.

## Why It's Good For The Game

Anti-salt measures.

## Changelog
:cl:
add: Dchat can now be toggled off/on by admins.
add: Buildmode now has an offer control mode.
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/admin/admin.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+/datum/admins/proc/toggledchat()
+	set category = "Server"
+	set desc="Toggle dis bitch"
+	set name="Toggle Dead Chat"
+	toggle_dchat()
+	log_admin("[key_name(usr)] toggled OOC.")
+	message_admins("[key_name_admin(usr)] toggled OOC.")
+	SSblackbox.record_feedback("nested tally", "admin_toggle", 1, list("Toggle OOC", "[GLOB.ooc_allowed ? "Enabled" : "Disabled"]")) //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/modules/admin/admin.dm]: Shouldn't it say deadchat instead of OOC?

Comment (Gandalf2k15) [reply modular_skyrat/master_files/code/modules/admin/admin.dm]: refresh the page

File: modular_skyrat/master_files/code/modules/admin/admin.dm [hunk 2]
```diff
@@ -0,0 +1,20 @@
+GLOBAL_VAR_INIT(dchat_allowed, TRUE)
+
+/datum/admins/proc/toggledchat()
+	set category = "Server"
+	set desc="Toggle dis bitch"
+	set name="Toggle Dead Chat"
+	toggle_dchat()
+	log_admin("[key_name(usr)] toggled dead chat.")
+	message_admins("[key_name_admin(usr)] toggled dead chat.")
```
Comment (tf-4) [inline modular_skyrat/master_files/code/modules/admin/admin.dm:9]: OOC?

**Comment by Useroth** [comment]
`set desc="Toggle dis bitch"` tho

but the PR is gud

**Pull request merged**